docs/sources/writing-guide/documentation-structure/index.md

```diff
@@ -12,21 +12,52 @@ keywords:
 
 # Documentation structure
 
-The technical writing team makes very intentional decisions about how we organize and structure product documentation. Users can find what they need quickly and easily with well-structured content.
+The Grafana Labs Docs team makes intentional decisions about how to organize and structure product documentation. With well-structured content, users can find what they need quickly and easily.
 
-Information architecture (IA) refers to the way in which content is organized. Before you start contributing to product documentation, it is important to understand the IA of the content.
+Before you begin contributing to documentation, it is important to understand the structure of the content.
 
-Generally, the IA determines how content is:
+According to STC’s [Information Design and Information Architecture: Why Technical Communicators Should Care About These Fields](https://www.stc.org/intercom/2022/05/information-design-and-information-architecture-why-technical-communicators-should-care-about-these-fields/),
+information architecture is "...the practice of organizing, structuring, and labeling information to it’s easy to find, use, and understand..."
+
+Generally, documentation structure determines how content is:
 
 - Titled
 - Grouped
 - Combined (or not combined) with other, related content
 
 The following examples are based on the Grafana OSS and Enterprise documentation.
 
-## Information architecture
+## Structure of published content
+
+Using the same content structure across documentation provides consistent experience. Topics are listed from high level to more specific. For example, a new Grafana user may wish to learn conceptual information first, so Introduction is listed before installation.
+
+Nat all Topics are not used for every product. For example, Grafana OSS may use all of the headings, while Grafana Enterprise Traces only uses a subset.
+
+This table provides a list of the high-level topics used for grouping content. For example, a conceptual page explaining metrics would go under the Introduction topic.
+
+Italicized topics are optional and are usually found in specific contexts. For example, the Create, Manage, Monitor topics are used in Grafana OnCall but are not used in Grafana Tempo.
+
+| Topic | Example link | Contains |
+| --- | --- | --- |
+| Introduction | Introduction to Grafana | Conceptual information, fundamentals, architecture, etc. |
+| Get started | Get started with Tempo | Opinionated walk-throughs and tutorials |
+| Set up | Set up Loki | System requirements, Set up, configure, upgrade, migrate, etc. |
```

Comment:
```suggestion
| Set up | Set up Loki | System requirements, and subsections titled _Set up_, _Configure_, _Upgrade_, or _Migrate_. |
```

Reply:
How do I implement the product?
Includes both install and configure 
User journey: Implementation and adoption

Reply:
See #210.